Freshli.Web/Models/ErrorViewModel.cs

```diff
@@ -0,0 +1,9 @@
+using System;
+
+namespace Freshli.Web.Models {
+  public class ErrorViewModel {
+    public string RequestId { get; set; }
+
+    public bool ShowRequestId => !string.IsNullOrEmpty(RequestId);
```

Comment:
Does `RequestIdIsSet` make more sense?

Reply:
This was autogenerated by the `dotnet new mvc` command. I'm not a fan of naming boolean fields like that. Sounds like a method name. I'll fix this.

Reply:
Created issue #59.